developer/server/build.sh

```diff
@@ -0,0 +1,192 @@
+#!/bin/bash
```

Comment:
Do you want this to also be:
```suggestion
#!/usr/bin/env bash
```
?